edu.cuny.hunter.log.ui/src/edu/cuny/hunter/log/ui/wizards/LogWizard.java

```diff
@@ -142,9 +144,16 @@ public void createControl(Composite parent) {
 
 			// set up buttons.
 			Button checkButton4 = this.addBooleanButton(
+					"Never lower the logging level of logging statements with particular keywords in their messages.",
+					NOT_LOWER_LOG_LEVEL_KEY_WORDS, this.getProcessor()::setNotLowerLogLevelWithKeyWords, result,
+					SWT.CHECK);
+			checkButton4.setSelection(true);
```

Comment:
These should be read from a store. That's the problem with the other bug. https://github.com/ponder-lab/Optimize-Java-8-Streams-Refactoring/blob/a648cce999a36c698aef520e9d8ce516cd50301e/edu.cuny.hunter.streamrefactoring.ui/src/edu/cuny/hunter/streamrefactoring/ui/wizards/OptimizeStreamRefactoringWizard.java#L74